packages/jsii/test/negatives/neg.submodules-cannot-have-colliding-names.ts

```diff
@@ -0,0 +1,7 @@
+///!MATCH_ERROR: Submodule "ns1" conflicts with "Ns1".
+
+export * as ns1 from './namespaced';
+
+export class Ns1 {
```

Comment:
Would `my_module` and `myModule` collide as well?

Reply:
Yes. I guess I could add more coverage on this... Initially I only guarded against `PascalCase` / `camelCase` collisions, but figured Java & Python would use `snake_case`.